src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -3,7 +3,9 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
```

Comment:
>IO [](http://example.com/codeflow?start=13&length=2)

Do you really need these 2 namespaces? From the code it seems there is no need. Could you confirm?